postgres/datadog_checks/postgres/postgres.py

```diff
@@ -126,10 +132,10 @@ def _run_query_scope(self, cursor, scope, is_custom_metrics, relations_config, c
 
             results = cursor.fetchall()
         except psycopg2.errors.FeatureNotSupported as e:
-            # This happens for example when trying to get replication metrics
-            # from readers in Aurora. Let's ignore it.
+            # This happens for example when trying to get replication metrics from readers in Aurora. Let's ignore it.
             log_func(e)
             self.db.rollback()
+            self._is_aurora = None
```

Comment:
Why do we need to reset `self._is_aurora` here ?

Reply:
Cause this error happens when trying to run things in aurora that we shouldnt so we have it wrong

Reply:
@hithwen Thx. 

(nit) Could you add a comment about why we reset `self._is_aurora` here. Will be helpful to readers.

Reply:
there is already a comment two lines above explaining that the error happens when trying to run things on aurora that you cannot

Reply:
yes, saw that, but was thinking it might be more explicit.